<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlCenter</class>
 <widget class="QWidget" name="ControlCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMP_hack Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="clientList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clientCreate">
       <property name="text">
        <string>Create Client</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clientDelete">
       <property name="text">
        <string>Delete Client</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scriptRun">
       <property name="text">
        <string>Run Script</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="scriptLabel">
       <property name="text">
        <string>Script:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scriptPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="scriptBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientList</tabstop>
  <tabstop>clientCreate</tabstop>
  <tabstop>clientDelete</tabstop>
  <tabstop>scriptRun</tabstop>
  <tabstop>scriptPath</tabstop>
  <tabstop>scriptBrowse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
